src/QuizPage/QuizPage.js

```diff
@@ -0,0 +1,129 @@
+import React, { Fragment, Component } from 'react';
+import styled from 'styled-components';
+import Answer from './Answer';
+import Question from './Question';
+
+const StyledButton = styled.div`
+    border: 2px solid #0094da;
+    border-radius: 0;
+    background-color: #0094da;
+    color: #fff;
+    display: inline-block;
+    font-size: 16px;
+    font-weight: 600;
+    padding: 14px 75px;
+    margin: 0 auto;
+    text-transform: uppercase;
+    transition: color .2s ease, background-color .2s ease;
+    cursor: pointer;
+
+    &:hover {
+        background-color: #fff;
+        color: #0094da;
+      }
+    
+`;
+
+const StyledAnswers = styled.div`
+      display: flex;
+      flex-wrap: wrap;
+      margin-top: 10px;
+      justif
+`;
+
+class QuizPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            questionCount: 0,
+            showButton: false,
+            isAnswered: false,
+            hasEnded: false,
+            score: 0,
+        }
+    }
+
+    showButtonHandler = () => {
+        this.setState({
+            isAnswered: true,
+            showButton: true
+        });
+    }
+
+    nextStep = () => {
+        const { questionCount} = this.state;
+        if (questionCount+1 === this.props.quizData.length) {
+            this.props.hasEnded(true);
+            this.setState({
+                hasEnded: true
```

Comment:
score and hasEnded is double stored in this component state and an upper component state as well, that's completely unnecessary, causing confusion and introduces the possibility of failing to synch them. 
the prop name hasEnded suggests it's a boolean, but in reality it is a function, which I guess sets the above mentioned state, proper naming is an important aspect of clean and understandable code